main.tf

```diff
@@ -146,6 +146,13 @@ resource "aws_ecs_task_definition" "fargate_task" {
   cpu                      = var.task_cpu
   memory                   = var.task_memory
 
+  dynamic "ephemeral_storage" {
+    for_each = var.ephemeral_storage > 20 ? [var.ephemeral_storage] : []
```

Comment:
if you use `ephemeral_storage=10` won't work, consider to use `var.ephemeral_storage > 0`

Reply:
hello @nicor88 
I'm using terraform 1.0.6 + aws provider 3.64.0 and didn't see the error.  Can I see the error message to reproduce? thanks.

Reply:
there is not error on terraform itself, it's a semantical error.

Reply:
https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_EphemeralStorage.html
```
sizeInGiB
The total amount, in GiB, of ephemeral storage to set for the task. The minimum supported value is 21 GiB and the maximum supported value is 200 GiB.
```
If I set ephemeral_storage=10, I got 
```
Error: expected ephemeral_storage.0.size_in_gib to be in the range (21 - 200), got 10
```
since the parameter is designed to enlarge instead of adding another.

Reply:
ok, then ignore my message. looks good then:)